src/view-slot.js

```diff
@@ -339,7 +339,9 @@ export class ViewSlot {
 
       if (returnToCache) {
         for (i = 0; i < ii; ++i) {
-          children[i].returnToCache();
```

Comment:
Let's eliminate the double lookup by storing the child in a local variable so we don't have to index it again if it is defined.